play/src/lib/styledcomponents/StyledComponents.tsx

```diff
@@ -97,12 +97,46 @@ export const IntroTextField = styled(TextField)(({ theme }) => ({
 }));
 
 export const JoinGameBackgroundContainer = styled(Stack)(({ theme }) => ({
-  height: '100%',
+  height: '100svh',
   position: 'fixed', // 100%, fixed to prevent sizing changes on mobile based on url bar etc
   width: '100%',
   background: `${theme.palette.primary.radialGradient} no-repeat`,
   backgroundSize: `100%`,
-  backgroundPosition: 'center',
+  backgroundPosition: 'center top',
+  backgroundAttachment: 'fixed',
   display: 'flex',
   alignItems: 'center',
 }));
+
+// icons for the avatar select screen
+interface AvatarIconProps {
+  isSelected: boolean;
+}
+
+export const AvatarIcon = styled('img', {
```

Comment:
General question on this file, any reason we're adding all in one file instead of breaking it into separate files?

Reply:
The idea on this file is that it acts as a library for the different styles. So I am thinking about it more as though it's like a css page instead of a series of components. That being said, as I write more pages it's bloating a lot and definitely needs to be broken down. 

Do you think it would make more sense to extract each component here (similar to a standard React structure) or should they be grouped by use cases (inputs, layout items, etc)? I think it might make sense to just extract each component (they're called styled components after all) but it will grow the number of files dramatically.

Reply:
Hmm, I think it depends on how large this file becomes and how many more styles we're going to add. Having too many files wouldn't be that bad as we can always use VSCode to just jump to a file and if we follow the pattern similar to ReactJS, that means by default, you know that `BodyCard` style is in `BodyCard` under `StyledComponents`. If we go this path, you probably want to start adding `Style` suffix to all of these. This way, it's obvious it's just CSS styling and not a component. This way, you can have `BodyCard` reactjs component and `BodyCardStyle`  for CSS part

Reply:
Yeah, agreed. I switched over to this now and it seems a lot cleaner. One note, instead of `Style` I've used `Styled` as we end up using the styled component like `<StyledComponent />`. If we went with `Style` we would have `<ComponentStyle />` which felt kind of odd to me, so I'm doing `<ComponentStyled />` instead. Still feels clunky but communicates that it's a styled component at least. Let me know if you have a better way of doing this though

Reply:
Sounds good thanks! I think as you're working mainly on this, see how it works out and if still didn't feel good or too much work to do it this way, we can revisit and find alternatives